assets/scss/components/stencil/navPages/_navPages.scss

```diff
@@ -320,7 +321,7 @@
         @include breakpoint("medium") {
             border-left: 1px solid stencilColor("navPages-subMenu-separatorColor");
             display: inherit;
-            height: $navPage-subMenu-horizontal-height;
```

Comment:
I didn't see that it was being used anywhere else but was hesitant to remove it myself. If you think it's best, I can go ahead with it.